### ISSUE TITLE
Why the loss is negative: -torch.mean((ent_x + ent_y) / ent_joint)

### ISSUE THREAD

**Issue by wangbm** [qiuhuaqi/midir#11]
https://github.com/qiuhuaqi/midir/blob/ea6e96e7df6d1f06cc2c69b1d4cb6c972b5c0d15/model/loss.py#L122

**Comment by wangbm** [comment]
Hi huaqi, When I remove the negative sign, MI loss is a positive number like the general training loss, and it can converge and decline normally at first glance, but from the verification metric dice, there is a problem. When I keep MI loss as a negative number and loss as a negative number, the loss attenuation fluctuation curve is strange, but the verification effect is normal. I want this loss to be a positive number like normal training, and ensure that the model is trained and verified normally. Is there any solution? thanks
